Updated UI to support all transaction types

blubaugha opened the pull request:
### Issues Fixed
UI part of issue: https://github.com/JumpMind/commerce/issues/2243

### Summary
Updates the UI to support showing all transaction types.

Currently the UI will still only show **Sales** and **Returns**. I changed the database data to get the different types shown in the screenshots.

The backend change for this will come next.

### Screenshots
**Desktop**
![trans-search--desktop](https://user-images.githubusercontent.com/3991426/96953230-991ba680-14be-11eb-9b86-3253237dad9b.gif)

**ELO POS - Landscape**
![trans-search--elopos-landscape](https://user-images.githubusercontent.com/3991426/96953262-a59fff00-14be-11eb-9055-d3f7874d9550.gif)

**ELO POS - Portrait**
![trans-search--elopos-portrait](https://user-images.githubusercontent.com/3991426/96953283-b5b7de80-14be-11eb-9117-863a58003bb3.gif)

**IPad Pro - Landscape**
![trans-search--ipadpro-landscape](https://user-images.githubusercontent.com/3991426/96953308-c10b0a00-14be-11eb-8e46-70c22dc41159.gif)

**IPad Pro - Portrait**
![trans-search--ipadpro-portrait](https://user-images.githubusercontent.com/3991426/96953324-c8caae80-14be-11eb-99a9-e72f04ec3e07.gif)

**IPad - Landscape**
![trans-search--ipad-landscape](https://user-images.githubusercontent.com/3991426/96953345-d54f0700-14be-11eb-94e9-e542978e540c.gif)

**IPad - Portrait**
![trans-search--ipad-portrait](https://user-images.githubusercontent.com/3991426/96953362-e13ac900-14be-11eb-94a1-d0237a11fe7d.gif)

mmichalek reviewed (commented):
Andy, I reviewed the changes with our newly formed design review board (Klem, Jen, and I) and we love them.  One comment: can we go with grey like this:
![image](https://user-images.githubusercontent.com/4398634/97030609-00703f80-152d-11eb-9bc3-05b7f7506e6b.png)

mmichalek reviewed (commented):
What about this for a suspend icon?
![image](https://user-images.githubusercontent.com/4398634/97030963-8f7d5780-152d-11eb-950d-e43fe498217f.png)

blubaugha commented:
> Andy, I reviewed the changes with our newly formed design review board (Klem, Jen, and I - :) ) and we love them. One comment: can we go with grey like this:
> ![image](https://user-images.githubusercontent.com/4398634/97030609-00703f80-152d-11eb-9bc3-05b7f7506e6b.png)

Hey @mmichalek , I have some updates for the design review board to review :)

1. Updated icon color to match gray text color
2. Updated icon for `SUSPEND_RECEIVED` and `SUSPEND_CANCELLED` transaction statuses to match the **Pause** button icon used for the `SUSPENDED` transaction status icon
